intera_interface/src/intera_interface/camera.py

```diff
@@ -110,6 +110,9 @@ def set_callback(self, camera_name, callback, callback_args=None,
         @param callback_args: additional arguments to pass to the callback
         @type queue_size: int
         @param queue_size: maximum number of messages to receive at a time
+        @type buff_size: int
+        @param buff_size: incoming message buffer size in bytes.
+            Defaults to handle head image of 4.1 MB.
```

Comment:
Technically it's 16 MB  (2^24 / 2^20).

Miao set that to give plenty of buffer margin, though it would probably be fine at 2^23.